src/strategy.ts

```diff
@@ -115,6 +115,9 @@ export abstract class Strategy {
   ): Promise<Update[]>;
 
   async getComponent(): Promise<string | undefined> {
+    if (!this.includeComponentInTag) {
```

Comment:
I'm glad this can be fixed at the `Strategy` level, and that we avoid custom logic in the Node strategy.